apps/src/lib/node/ledger/shims/abcipp_shim.rs

```diff
@@ -8,7 +8,7 @@ use futures::future::FutureExt;
 use namada::types::ethereum_events::EthereumEvent;
 #[cfg(not(feature = "abcipp"))]
 use namada::types::hash::Hash;
-use tokio::sync::mpsc::{UnboundedReceiver, UnboundedSender};
+use tokio::sync::mpsc::{Receiver as BoundedReceiver, UnboundedSender};
```

Comment:
nit: we can just import `Receiver` as normal, we should get used to interpreting `Sender`/`Receiver` as bounded. We are importing it as `Receiver` in `shell/mod.rs`